XenAdmin/TabPages/VMStoragePage.resx

```diff
@@ -421,13 +421,13 @@
     <value>1</value>
   </data>
   <data name="DetachButtonContainer.Location" type="System.Drawing.Point, System.Drawing">
-    <value>602, 2</value>

```

Comment:
I haven't looked at the control, but this line of the diff looks suspicious. It seems one of the buttons has moved a long distance.